src/work-allocation-2/models/cases/case-service-config.model.ts

```diff
@@ -0,0 +1,9 @@
+import { CaseFieldConfig } from '.';
+import { CaseService, SortOrder } from './../../enums';
+
+export default interface CaseServiceConfig {
```

Comment:
_Question:_
Is this identical to `TaskServiceConfig `? Is there scope for a more generic `WorkServiceConfig `?

Reply:
I honestly tried doing this however after making changes to 11 files I searched for TaskServiceConfig and it would involve changing a significant number of files. Suggest re-addressing this at a later date (IP week?)